test/new-e2e/agent-platform/common/agent_behaviour.go

```diff
@@ -0,0 +1,203 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
Can you move `agent-platform/*` to `tests/agent-platform/*` ?

Reply:
Done!